satpy/tests/reader_tests/modis_tests/_modis_fixtures.py

```diff
@@ -228,9 +228,7 @@ def generate_imapp_filename(suffix):
 
 def create_hdfeos_test_file(filename: str,
                             variable_infos: dict,
-                            geo_resolution: Optional[int] = None,
-                            file_shortname: Optional[str] = None,
-                            include_metadata: bool = True):
+                            metadata_dict: Optional[dict] = {}):
```

Comment:
No mutable default arguments please.

```suggestion
                            metadata_dict: Optional[dict] = None):
```